extras/applications/FaceAndEyesDetection/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
```

Comment:
It seems that pipenv cannot install OpenCV automatically. If you think it's more appropriate to use distro-provided packages, you can delete the Pipfile and add instructions on installing `python3-opencv` and other necessary packages.